cores/esp8266/umm_malloc/umm_malloc.cpp

```diff
@@ -495,6 +495,7 @@
 #include <string.h>
 #include <pgmspace.h>
 
+#define _UMM_MALLOC_CPP
```

Comment:
Drop this define, as mentioned elsewhere.

Reply:
Done.